src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -41,9 +46,23 @@ class ConfirmationPage extends React.Component {
     const response = form.submission.response
       ? form.submission.response.attributes
       : {};
-    const { veteranFullName, verified } = form.data;
+    const {
+      first: firstName = '',
+      middle: middleName = '',
+      last: lastName = '',
+      suffix = ''
+    } = form.data.veteranFullName;
+
+    const { serviceBranch, verified } = form.data;
+
+    const photoUrl = getImageUrl(form.data.photo);
+
     const submittedAt = moment(form.submission.submittedAt);
 
+    const veteranFullNameStr =
+      `${firstName.toUpperCase()} ${middleName.toUpperCase()} ${lastName.toUpperCase()} ${suffix.toUpperCase()}` // upper case name
```

Comment:
Opted for passing the whole name string in rather than the name parts even though the veteranID card breaks them back down into parts. The number of name parts vary (2 minimum up to 3 typically but can be higher with space separated double-barrel last names) so I thought it'd be simpler if the ID component could just assume a space splittable string since the which name part they are isn't relevant to it.